index.html

```diff
@@ -25,10 +25,19 @@
 
             <div class="row right" class="right hide-on-med-and-down">
                 <a href="./SigninAuth.html" class="btn-large waves-effect waves-light deep-orange" id="login-btn">User
-
                     Login</a>
             </div>
+
+            <ul id="dropdown1" class="dropdown-content">
+                <li id="one"><a href="#!">one</a></li>
+                <li id="two"><a href="#!">two</a></li>
+                <li id="three"><a href="#!">three</a></li>
+                <li id="four"><a href="#!">four</a></li>
+                <li id="five"><a href="#!">five</a></li>
+            </ul>
             <ul id="nav-mobile" class="right hide-on-med-and-down">
+                <li><a class="dropdown-trigger" href="#!" data-target="dropdown1">View Tokens<i
```

Comment:
While you're fixing other issues: the href should just be href="#", and no exclamation mark. all 6 of these above